src/components/book.js

```diff
@@ -1,21 +1,15 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
[REQUIRED] I have noticed you disabled eslint it is against microverse policy to disable linter because linters help us write clean code. Please kindly enable eslint.